en/08_Changelogs/alpha/6.0.0-alpha1.md

```diff
@@ -2401,7 +2406,7 @@ If you were calling `getSchemaData()` in your `getAttributes()` method in a `For
   - 2024-09-10 [a0ad75397](https://github.com/silverstripe/silverstripe-framework/commit/a0ad753974b5904746e05b4a028b0caeedfb0ceb) Create DBClassNameVarchar (Steve Boyd)
   - 2024-08-25 [e3508d41d](https://github.com/silverstripe/silverstripe-framework/commit/e3508d41d520983262e2fba28f7b6db0785e6d72) Provide a standardised CMSEditLink method (#11338) (Guy Sartorelli)
   - 2024-08-15 [64a17e09d](https://github.com/silverstripe/silverstripe-framework/commit/64a17e09d4eb66da7e17902d6c0f7d9b0e2d2340) Various changes to support SiteTree form field scaffolding (#11327) (Guy Sartorelli)
-  - 2024-07-01 [98dc238d2](https://github.com/silverstripe/silverstripe-framework/commit/98dc238d2aec1558a604941f6fc608c789702e03) Do not require _config dir or _config.php for modules (Steve Boyd)
+  - 2024-07-01 [98dc238d2](https://github.com/silverstripe/silverstripe-framework/commit/98dc238d2aec1558a604941f6fc608c789702e03) Do not require _config dir or `_config.php` for modules (Steve Boyd)
```

Comment:
Needed to avoid `_` being seen as a markdown character. Will update the script to avoid this in the future.